test/utils.js

```diff
@@ -343,3 +344,61 @@ describe('Apify.utils.sleep()', () => {
             });
     });
 });
+
+describe('utils.extractUrls()', () => {
```

Comment:
It would be good idea to test also invalid URLs to ensure they are not matched, plus try more complicated strings like `http://www.example1.com,http://www.example2.com,http://,http://`, test URLs with special characters and escapes etc.

Reply:
Yes, I plan to add more test cases as I find more suitable URLs.

Just saving this list for later use: https://cran.r-project.org/web/packages/rex/vignettes/url_parsing.html